### PR TITLE
ECL-339: ECL Account - View your returns

### PR DESCRIPTION
**Pull request by nemanjatomanovic**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -12,4 +12,5 @@ GET        /account/sign-out-survey            uk.gov.hmrc.economiccrimelevyacco
 GET        /account/sign-out                   uk.gov.hmrc.economiccrimelevyaccount.controllers.SignOutController.signOutNoSurvey()
 GET        /account/signed-out                 uk.gov.hmrc.economiccrimelevyaccount.controllers.SignOutController.signedOut()
 
+GET        /account/view-returns               uk.gov.hmrc.economiccrimelevyaccount.controllers.ViewYourReturnsController.onPageLoad()
```
Comment (TonyCOgilvie) [inline conf/app.routes:15]: An integration test is missing for this route, i.e. ViewYourReturnsISpec?

File: app/uk/gov/hmrc/economiccrimelevyaccount/views/ReturnsView.scala.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    }
+}
+@layout(
+    pageTitle = title(messages("account.title")),
+    showBackLink = true,
+    fullWidthContent = false
+) {
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-0">@messages("account.viewReturns.title")</h1>
+
+    <br>
```
Comment (TonyCOgilvie) [inline app/uk/gov/hmrc/economiccrimelevyaccount/views/ReturnsView.scala.html:74]: Strictly speaking empty elements such as this should be <br/> (shorthand for <br></br>)

**Review by TonyCOgilvie — COMMENTED** [review]
A couple of comments added.

File: app/uk/gov/hmrc/economiccrimelevyaccount/views/ReturnsView.scala.html (hunk 2 above)
Comment (nemanjatomanovic) [reply app/uk/gov/hmrc/economiccrimelevyaccount/views/ReturnsView.scala.html]: Done 😄

File: conf/app.routes (hunk 1 above)
Comment (nemanjatomanovic) [reply conf/app.routes:15]: Done 😄

**Review by TonyCOgilvie — APPROVED** [review]

**Pull request merged**